File: 深化/_2444.java
import java.util.Scanner;

public class _2444 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int input = sc.nextInt();

        for (int i = 0; i < input; i++) {
            for (int j = 0; j < input - i - 1; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < i * 2 + 1; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
        for (int i = 0; i < input - 1; i++) {
            for (int j = 0; j < i + 1; j++) {
                System.out.print(" ");
            }
            for (int j = 0; j < (input - i - 1) * 2 - 1; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: 06. Deepening 1/_2444.py
N = int(input())

for i in range(0, N):
    for j in range(N - 1 - i):
        print(end=" ")
    for j in range(i * 2 + 1):
        print("*", end="")
    print()

for i in range(0, N - 1):
    for j in range(i + 1):
        print(end=" ")
    for j in range(N * 2 - 3 - i * 2):
        print("*", end="")
    print()